testutil/daemon/daemon.go

```diff
@@ -45,6 +45,12 @@ type logT interface {
 	Logf(string, ...interface{})
 }
 
+// nopLog is a no-op implementation of logT that is used in daemons created by
+// NewDaemon (where no testingT is available).
+type nopLog struct{}
```

Comment:
I'm not sure we want to do a noop when logging, it's useful to know what's going on when starting daemons. @tonistiigi wdyt?

Reply:
This is going to be used from buildkit. At least at first it is just in tests, but I'd hate for it to do its own logging that differs from whatever buildkit does. As a general rule of thumb I try to avoid logging from within libraries, even if it's just for testing.

Reply:
Would it be worth adding an option to set the logger? This could be the default but you could override it if you wanted logging so that in buildkit we could log however it normally does things.